near-sdk/src/utils/mod.rs

```diff
@@ -0,0 +1,30 @@
+#[macro_export]
+macro_rules! log {
+    ($arg:tt) => {
+        crate::env::log($arg.as_bytes())
+    };
+    ($($arg:tt)*) => {
+        crate::env::log(format!($($arg)*).as_bytes())
```

Comment:
`format!` is known for huge code size: https://jamesmunns.com/blog/fmt-unreasonably-expensive/

Have you checked the impact on our contracts?

Reply:
Since this a macro, it doesn't increase the code unless someone uses it. Which is just a in-place replacement for `env::log(format!(...).as_bytes())` until we start focusing on `no_std`

Reply:
E.g. `status_message.wasm` didn't increase as much (+362 bytes after using `log!`):